src/Servers/test/FunctionalTests/NoCompression.conf

```diff
@@ -14,6 +14,15 @@ http {
     keepalive_timeout 10;
     types_hash_max_size 2048;
 
+    # nginx fails start on Fedora/RHEL as non-root with permission errors when
+    # trying to create these paths at their default location under /var/lib/nginx/tmp.
+    # Set them to a writable location.
+    client_body_temp_path /tmp/client_body;
```

Comment:
can `/tmp/` always be assumed -- eg., $TMPDIR may be set? (but, I guess it's not a goal for this script to work every possible place)

Reply:
On Unix, this should be fine.
Probably these paths will be an issue on Windows.
Let's see what CI makes of it.

Reply:
CI doesn't run `nginx` tests on Windows; we don't even have infrastructure for that case (though it's possible to write). 
Testing on macOS will be more important.

Reply:
And it appears the build agent-hosted macOS tests (which should include these tests) worked fine. Please confirm the tests actually ran in that job; many tests in this project are conditional or quarantined.